<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arduino Relay Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDeviceAddress">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>IP address. Received by broadcast and should be filled in automatically</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRelay1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Relay 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRelay2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Relay 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRelay3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Relay 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRelay4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Relay 4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDeviceMAC">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn250ms">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Close relay 1 for 250ms</string>
    </property>
    <property name="text">
     <string>250ms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn500ms">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Close relay 1 for 500ms</string>
    </property>
    <property name="text">
     <string>500ms</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
